rust/migration/src/m20231127_232833_drop_use_time.rs

```diff
@@ -0,0 +1,40 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(JobUses::Table)
+                    .drop_column(JobUses::Time)
+                    .to_owned(),
+            )
+            .await
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(JobUses::Table)
+                    .add_column(
+                        ColumnDef::new(JobUses::Time)
+                            .timestamp()
+                            .not_null()
+                            .default(SimpleExpr::Keyword(Keyword::CurrentTimestamp)),
```

Comment:
This should be replaced with CreateTime not the current time stamp

Reply:
lol how do you do that?

Reply:
I don't know lol